modules/administration/pages/backup-restore.adoc

```diff
@@ -432,24 +446,23 @@ Start the spacewalk services with:
 
 
 This section provides guidance on restoring your server after its root partition has crashed.
-This section assumes you have setup your server similar to the procedure explained in Getting Started guide with separate partitions for the database and for channels mounted at [path]``/var/lib/pgsql``
- and [path]``/var/spacewalk/``
-.
+This section assumes you have setup your server similar to the procedure explained in Installation guide with separate partitions for the database and for channels mounted at [path]``/var/lib/pgsql`` and [path]``/var/spacewalk/``.
 
 .Procedure: Recovering from a Crashed Root Partition
-. Start by installing SLES12 SP2 and the SUSE Manager Extension. Do not mount the [path]``/var/spacewalk`` and [path]``/var/lib/pgsql`` partitions.
-. Once installation of SUSE Manager has completed shutdown services with [command]``spacewalk-service shutdown`` and the database with [command]``rcpostgresql stop``.
-. Mount your [path]``/var/spacewalk`` and [path]``/var/lib/pgsql`` partitions and restore the directories listed in section one.
-. Start SUSE Manager services and the database with [command]``spacewalk-services start`` and [command]``rcpostgresql start``
-. SUSE Manager should now operate normally without loss of your database or synced channels.
+. Start by installing {productname}.
+Do not mount the [path]``/var/spacewalk`` and [path]``/var/lib/pgsql`` partitions.
+. When the installation of {productname} has completed shutdown the services with [command]``spacewalk-service shutdown`` and the database with [command]``rcpostgresql stop``.
+. Mount your [path]``/var/spacewalk`` and [path]``/var/lib/pgsql`` partitions and restore the directories listed in <<backup-restore-product>>.
+. Start {productname} services and the database with [command]``spacewalk-services start`` and [command]``rcpostgresql start``.
+. {productname} should now operate normally without loss of your database or synced channels.
```

Comment:
Delete this line, it's a result statement, not a step.

Reply:
ok, it's no longer a step.